### DISCUSSION TITLE
Loops or multiple item fields?

### DISCUSSION THREAD

**Discussion by ahme-dev** [pdfme/pdfme#167] [Q&A] [answered]
Is it possible to have a field with looping data or conditional data.

Something like: 

{{ skills }}

Into:

- Skill1
- Skill2
- Skill3

**Comment by hand-dot** [comment, marked as answer]
@ahmeddots 
No, we can't

**Reply by ahme-dev** [reply]
Cool library, nonetheless. Keep up the good work!

Any workarounds for now?

So I have to make a field for each item and number them like:

{{ skill1 }}
{{ skill2 }}
{{ skill3 }}

And if I don't fill a field, the field-place will be empty?

**Reply by hand-dot** [reply]
I am also currently using that workaround. The following StackOverflow link might be useful.

https://stackoverflow.com/questions/71563736/grid-like-labels-stickers-dynamically-generating-node-js

**Reply by ahme-dev** [reply]
Thanks for the timely answers.

This project is only one I've found to fit my team's needs, so we're really grateful it exists, and hope to see it thrive.

We also hope to contribute in the future.

**Comment by jsgosh** [comment]
Related issue: #671
